pkg/service/did/resolve/universal.go

```diff
@@ -0,0 +1,49 @@
+package resolve
+
+import (
+	"bufio"
+	"io"
+	"net/http"
+
+	didsdk "github.com/TBD54566975/ssi-sdk/did"
+	"github.com/goccy/go-json"
+	"github.com/pkg/errors"
+)
+
+type UniversalResolver struct {
+	C   http.Client
```

Comment:
can we use 'client' instead of C unless this is a golang paradigm or something

Reply:
Done